src/otx/core/model/module/classification.py

```diff
@@ -56,12 +56,22 @@ def __init__(
             metric=metric,
         )
 
+    def configure_metric(self) -> None:
+        """Configure the metric."""
+        super().configure_metric()
+        if isinstance(self.metric, CustomAccuracy):
+            self.metric.label_info = self.model.label_info
+
     def _log_metrics(self, meter: Metric, key: str) -> None:
         results = meter.compute()
         if results is None:
             msg = f"{meter} has no data to compute metric or there is an error computing metric"
             raise RuntimeError(msg)
 
+        # Custom Accuracy returns the dictionary, and accuracy value is in the `accuracy` key.
+        if isinstance(results, dict):
+            results = torch.tensor(results["accuracy"])
```

Comment:
I wonder if the result always has 'accuracy', even when using other matrices.

Reply:
For the classification, currently yes. All custom matrices returns the accuracy key in the results. For the future implementation, uses need to returns the same way like this.